dscgt/recycling_website
# Record delete and update

Adds deletion and updating of checkout records and route records.

## How to test
1. Install new dependencies with `npm i`
1. Run the app with `npm start`
2. Use Firebase functions to add some checkout/checkin records, and also some route records to the emulated database
   - there are two endpoints available to seed some sample records; see the README for details
3. Update some records in the UI and ensure these changes show up in the emulated DB
4. Delete some records in the UI and ensure these changes show up in the emulated DB
